builtin/logical/ssh/path_sign.go

```diff
@@ -172,12 +213,28 @@ func (b *backend) pathSignCertificate(ctx context.Context, req *logical.Request,
 		return nil, errwrap.Wrapf("failed to parse stored CA private key: {{err}}", err)
 	}
 
+	var certValidAfter, certValidBefore uint64
+
+	// branch the logic for calculating the certificate ValidAfter & ValidBefore values
+	if validAfterWasSpecified && validBeforeWasSpecified {
+		// the ValidAfter/ValidBefore were specified as epoch's directly in the specified data
```

Comment:
Only apply the new logic to directly set ValidAfter/ValidBefore if they were both specified, we know that other logical checks already took place to ensure that `ttl` was not explicitly specified